e2e-tests/portal/cypress/support/e2e.js

```diff
@@ -3,3 +3,15 @@ import './commands';
 
 // Mitigates test fails due to js errors (third-party js)
 Cypress.on('uncaught:exception', () => false);
+
+// eslint-disable-next-line no-unused-vars
+Cypress.on('fail', (err, _runnable) => {
+  const pageBodyHtml = Cypress.$('body').html();
+
+  // Print the page body HTML to the console as part of the error if a test fails
+  // TODO: Remove this extra logging information when there are no more flakey e2e tests
```

Comment:
If we do raise a ticket for removing this logging, we should add the ticket number to all the TODOS